client/src/app/pages/vulnerability-list/vulnerability-context.tsx

```diff
@@ -87,8 +87,8 @@ export const VulnerabilitySearchProvider: React.FunctionComponent<
     getHubRequestParams({
       ...tableControlState,
       hubSortFieldKeys: {
+        severity: "severity",
```

Comment:
To sort by severity we could change this line to `severity: "average_severity"`.

The `hubSortFieldKeys` contains a key value object where the KEY is always the fields we defined in the UI. And the VALUE is the backend filter key we want to use.

Reply:
`severity: "average_score"` should also be equivalent to `severity: "average_severity"`

Reply:
ah I see, I'll get that updated now